src/Microsoft.DocAsCode.Build.TableOfContents/Constants.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.DocAsCode.Build.TableOfContents
 {
-    internal static class Constants
+    public static class Constants
```

Comment:
delete this class and move into common

---
In reply to: [121577705](https://github.com/dotnet/docfx/pull/1758#discussion_r121577705) [](ancestors = 121577705)